src/suggestions.js

```diff
@@ -113,7 +90,9 @@ export function getSuggestedQuery(element, variant = 'get', method) {
     })
   }
 
-  const labelText = getLabelTextFor(element)
+  const labelText = getLabels(document, element)
+    .map(label => label.content)
+    .join(' ')
```

Comment:
This looks like a behavior change. Is this a bug fix or a new feature?

Reply:
ooopss The space was a mistake

Reply:
ooopss The space was a mistake

Reply:
We should probably have a test for this. But still this looks like a change from the original behavior which is concerning as well. Maybe I'm missing something though.

Reply:
I have added the method `getLabels` that could return ore or more labels. Every label should be separated by space because they are regarding to different label object. I have added a test for it

Reply:
Should this fix the problem related to the suggestion in case of concatenated labels?

Reply:
Yes. I have added a test to prove this